charts/replicated-library/templates/_all.tpl

```diff
@@ -3,20 +3,18 @@ Main entrypoint for the replicated-library chart. It will render all underlying
 */}}
 {{- define "replicated-library.all" -}}
   {{- /* Merge the local chart values and the replicated-library chart defaults */ -}}
-  {{- include "replicated-library.values.setup" . }}
+  {{ include "replicated-library.values.setup" . }}
+  
+  {{- /* Create global context dicts */ -}}
+  {{- $_ := set $ "ContextNames" dict -}}
+  {{- $_ := set $ "ContextValues" dict -}}
 
   {{- /* Build the templates */ -}}
-
   {{ include "replicated-library.apps" . | nindent 0 }}
-
   {{ include "replicated-library.services" . | nindent 0 }}
-
   {{ include "replicated-library.ingresses" .  | nindent 0 }}
-
   {{ include "replicated-library.configmaps" . | nindent 0 }}
-
   {{ include "replicated-library.secrets" .  | nindent 0 }}
-
-  {{- include "replicated-library.pvc" . }}
+  {{ include "replicated-library.pvc" . }}
```

Comment:
I'm not sure why the `| nindent 0` is needed on the other sections but should this one have it as well?

Reply:
Guessing because this was the last set of templates included, so the PVCs will always be last in the list. But i'll make it consistent anyway, so someone else who adds a template afterwards doesn't get confused.